index.html

```diff
@@ -206,15 +206,15 @@ <h2 class="about-title">Skills</h2>
       <h2 class="contact-h2">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
       <form id="form"  action="https://formspree.io/f/mbjelbzj" method="POST">
         <div class="input-control">
-            <input id="fullname" name="fullname" type="text" placeholder="Full name">
+            <input id="fullname" name="fullname" type="text" placeholder="Full name" maxlength="30">
             <div class="error"></div>
         </div>
         <div class="input-control">
             <input id="email" name="email" type="text" placeholder="Email address">
             <div class="error"></div>
         </div>
         <div class="input-control">
-          <textarea name="message" id="message" cols="30" rows="10" placeholder="Enter text here"></textarea>
+          <textarea name="message" id="message" cols="30" rows="10" placeholder="Enter text here" maxlength="500"></textarea>
```

Comment:
- [ ] I was about to approve this project :clap: , this is a wonderful :pray:  However there is something you need to change. You should make sure that every section of your portfolio stick to the design it means even the position of the text should stick to the Figma design :100: . According, to picture below and kindly update your design :+1:  

### Your design

![Screenshot from 2023-01-23 23-42-04](https://user-images.githubusercontent.com/80458199/214157746-8bb3dea6-be4f-4b14-b2cf-90361e34d74a.png)

After adding a padding the following way, I got a correct design

```bash
padding: 50px;
```
### Figma design

![Screenshot from 2023-01-23 23-42-31](https://user-images.githubusercontent.com/80458199/214157930-60ee7090-6182-42ed-8d21-8bd271a07e6d.png)